### DISCUSSION TITLE
[Feature] user-wide default album sorting

### DISCUSSION THREAD

**Discussion by waclaw66** [immich-app/immich#8068] [Feature Request]
### The feature

It would be nice to have user-wide configuration for default album sorting. I personally prefer "oldest first" order for story-like albums (majority of my albums), however timelapse albums look better with "newest first" order.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by lobotomm** [comment]
I support this!

**Comment by Menghini** [comment]
I also agree with this. We should also dictate what order is default for shared albums as well

**Comment by yan-8** [comment]
I support this also!

**Comment by dagstuan** [comment]
I made a PR for this: #18950

**Comment by happychriss** [comment]
@dagstuan : Thank you :-) Works perfect!
